src/pineko/theory.py

```diff
@@ -395,8 +395,11 @@ def fk(self, name, grid_path, tcard, pdf):
         if sv_method is None:
             if not np.isclose(xif, 1.0):
                 check_scvar_evolve(grid, max_as, max_al, check.Scale.FACT)
-        # loading ekos
+        # loading ekos to produce a tmp copy
         with eko.EKO.edit(eko_filename) as operators:
```

Comment:
Here you do not need `edit` any longer, since you are only deep copying. You can just use `EKO.read`

Reply:
True! [8b2a634](https://github.com/NNPDF/pineko/pull/117/commits/8b2a63455585c8873980b958da7c1f3853d46cf2)